lib/jekyll/frontmatter_defaults.rb

```diff
@@ -130,7 +130,7 @@ def valid?(set)
     # new_scope - the new scope hash
     #
     # Returns true if the new scope has precedence over the older
-    def has_precedence?(old_scope, new_scope)
+    def precedence?(old_scope, new_scope)
```

Comment:
`precedence?` is [`private`](https://github.com/jekyll/jekyll/pull/4301/files#diff-68dac361a7950a8a653b234e362ba78aR79), so renaming is not a breaking change.